spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "Ownership" do
+  describe "reporting" do
+    describe "searching by current ldap group" do
+      let(:exp)    { {"="=>{"field"=>"Vm-owned_by_current_ldap_group", "value"=>"true"}} }
+      let(:report) { MiqReport.new.tap {|r| r.db = 'Vm'} }
+      let(:search_opts) { { :filter => MiqExpression.new(exp), :per_page => 20, :named_scope => :with_miq_group } }
+
+      before do
+        @miq_group = FactoryGirl.create :miq_group
+        User.current_user = FactoryGirl.create :user,
+                                               :current_group => @miq_group,
+                                               :miq_groups => [@miq_group]
+        expect(User).to receive(:server_timezone).and_return("UTC")
```

Comment:
I think you want [stub_user](https://github.com/ManageIQ/manageiq/blob/a8635229337db4fb077295a98da003258969da10/spec/support/auth_helper.rb#L14) since that creates the user, group, stubs current_user, does the `server_timezone` thing.  It returns the user...so you can do `user.miq_group_id` on line 15.

Note, the user_with_group factory doesn't do the timezone stubbing above, AFAIK.